lib/screens/about_page.dart

```diff
@@ -1,14 +1,23 @@
 import 'package:adaptive_theme/adaptive_theme.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 import 'package:smarthome/helper/helper_methods.dart';
 import 'package:smarthome/helper/theme_manager.dart';
 import 'package:smarthome/helper/update_manager.dart';
 import 'package:smarthome/models/version_and_url.dart';
 
-class AboutScreen extends StatelessWidget {
-  const AboutScreen({final Key? key}) : super(key: key);
+import '../helper/settings_manager.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
 
+class AboutPage extends ConsumerStatefulWidget {

```

Comment:
Ich sehe kein State in deinem Widget, dann könntest du eher `ConsumerWidget` verwenden